src/component/Book/Book.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Book({ title, author }) {
+  return (
+    <section>
+      <span>{title}</span>
+      <span>{author}</span>
+    </section>
+  );
+}
```

Comment:
- [ ] Following the project requirements, the Book component should have a button for removing books

* the button wouldn't remove the books *yet*, this is just a placeholder for now